packages/labs/compiler/test_files/handle_html_identifier_reassignment.js

```diff
@@ -0,0 +1,16 @@
+import {html as a} from 'lit';
+// TODO: Renaming import outside the import declaration is not supported.
+// This test doesn't currently work correctly.
```

Comment:
We miss this optimization, but it totally works!

Reply:
Oh I see! It "works" because it's still correct – just not optimized.

Great point – will reword.